aggregators/is-performant/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Aggregate = require('../aggregate');
```

Comment:
gatherer => audit => aggregate => USER :)

this layer of transformation seems "muddy" I want us to think about where this raw audit to human readable number / data transform happens.


I am not -1 to this idea, I just think that this could be named something more obvious